css/styles.scss

```diff
@@ -1,9 +1,100 @@
 @import "./variables/colors";
 @import "./variables/variables";
+@import "./variables/mixins";
 
 html,
 body {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  background-color: $grey;
+  font-family: "Titillium Web";
+}
+.container {
+  width: 30vw;
+  height: 95vh;
+  margin-top: 5%;
+  background-color: $white;
+  border-radius: 15px 15px 0 0;
+  .header {
+    @include flex($wrap: wrap, $align-content: space-between);
+    width: 30vw;
+    height: 30%;
+    background-size: cover;
+    background-image: url("../assets/background-2.jpg");
+    border-radius: 15px 15px 0 0;
+    .clear {
+      @include flex(flex-end, center);
+      width: 100%;
+      height: 4rem;
+    }
+    i {
+      margin-right: 1.5rem;
+      font-size: 20px;
+      color: $white;
+      cursor: pointer;
+    }
+    .date {
+      width: 100%;
+      height: 30px;
+      margin: 0 0 1rem 1rem;
+      color: $white;
+      font-size: 22px;
+    }
+  }
+  .content {
+    @include flex($direc: column);
```

Comment:
when applying mixin with include, you don't do `flex($direc: column)`, just `flex(column)`